pallets/vesting/src/benchmarking.rs

```diff
@@ -14,8 +14,6 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-#![cfg(feature = "runtime-benchmarks")]
```

Comment:
how come we don't need anymore? does it default to not compiling now?

Reply:
I moved it to the lib file to be consistent w/ the test and mock. I need to do a general cleanup of all the files like this but I was in here messing around trying to get something to compile so it caught my eye